VMBackup/main/ExtensionErrorCodeHelper.py

```diff
@@ -74,7 +75,8 @@ class ExtensionErrorCodeHelper:
 
             ExtensionErrorCodeEnum.error_http_failure : Status.ExtVmHealthStateEnum.red,
             ExtensionErrorCodeEnum.FailedRetryableSnapshotFailedRestrictedNetwork : Status.ExtVmHealthStateEnum.red,
-            ExtensionErrorCodeEnum.FailedRetryableSnapshotFailedNoNetwork : Status.ExtVmHealthStateEnum.red
+            ExtensionErrorCodeEnum.FailedRetryableSnapshotFailedNoNetwork : Status.ExtVmHealthStateEnum.red,
+            ExtensionErrorCodeEnum.FailedSnapshotLimitReached : Status.ExtVmHealthStateEnum.red
```

Comment:
check health state of this in Windows Extension, it might be yellow.

Reply:
it is red.
{ExtensionErrorCodeEnum.FailedSnapshotLimitReached, ExtVmHealthStateEnum.red}